client/src/client-only-routes/ShowProjectLinks.js

```diff
@@ -158,9 +171,16 @@ const ShowProjectLinks = props => {
   const { files, isOpen, projectTitle, solution } = solutionState;
   return (
     <div>
-      {certName === 'Legacy Full Stack'
-        ? `As part of this Legacy Full Stack certification, ${name} completed the following certifications:`
-        : `As part of this certification, ${name} built the following projects and got all automated test suites to pass:`}
+      <Trans
+        user={name}
+        i18nKey={
+          certName === 'Legacy Full Stack'
+            ? 'certification.project.heading-legacy-full-stack'
+            : 'certification.project.heading'
+        }
+      >
+        {{ user: name }}
+      </Trans>
```

Comment:
```suggestion
      {t(
        certName === 'Legacy Full Stack'
          ? 'certification.project.heading-legacy-full-stack'
          : 'certification.project.heading',
        { user: name }
      )}
```
Since it's just a string we can use `t` here, as Tom said.